stable.js

```diff
@@ -1099,6 +1099,25 @@ var idrinth = {
         },
         controls: null,
         tooltipTO: null,
+        buildModal:function(title,content,altFunc) {
```

Comment:
A simple example: buildModal(null, null, null) -> buildModal(undefined, undefined, undefined). Because API expose these as a public method in both scenarios popup should draw close sign, and have empty body and empty title.